src/S3/Transfer.php

```diff
@@ -198,7 +198,7 @@ private function determineScheme($path)
     }
 
     /**
-     * Normalize a path so that it has a trailing slash.
+     * Normalize a path so that it has UNIX-style directory separators and no trailing /
```

Comment:
This method does the opposite of the previous doc-comment. Stripping trailing / does appear to be important, however, as filesystem paths with trailing / produce unexpected results elsewhere in the class.